apps/wallet-mobile/src/yoroi-wallets/cardano/constants/cardano-config.ts

```diff
@@ -62,6 +62,7 @@ export const cardanoConfig = freeze(
             roles: {
               external: 0,
               internal: 1,
+              drep: 3,
```

Comment:
@michaeljscript byron doesn't support staking, so it won't support governance, if this if for typing only, buy checking if is 1852 will make TS infer the type.